README.md

```diff
@@ -52,7 +52,7 @@ Participation in the Kubernetes community is governed by the
 [sn]: https://github.com/kubernetes/community/tree/master/sig-network
 [cm]: https://kubernetes-sigs.github.io/gateway-api/community
 [slack]: https://kubernetes.slack.com/messages/sig-network-gateway-api
-[guides]: https://kubernetes-sigs.github.io/gateway-api/guides
+[getting-started]: https://kubernetes-sigs.github.io/gateway-api/getting-started
 [spec]: https://kubernetes-sigs.github.io/gateway-api/spec
 [concepts]: https://kubernetes-sigs.github.io/gateway-api/api-overview
 [security-model]: https://kubernetes-sigs.github.io/gateway-api/security-model
```

Comment:
Does not have to be fixed in this PR, but all of these should be updated to point to gateway-api.sigs.k8s.io.